Support/Global/correspondance.R

```diff
@@ -12,10 +12,7 @@ corresp <- function(df,
                    check_matches = FALSE
```

Comment:
The user now specifies a quoted character vector of variables over which to perform the pad function, such as c("var1", "isic") or NULL if no padding is desired